src/domain/service/noteSettings.ts

```diff
@@ -104,4 +104,19 @@ export default class NoteSettingsService {
   public async createTeamMember(team: TeamMemberCreationAttributes): Promise<TeamMember> {
     return await this.teamRepository.create(team);
   }
+
+  /**
+   * Updates invitation hash in note settings
+   *
+   * @param noteId - note internal id
+   * @returns updated note settings
+   */
+  public async regenerateInvitationHash(noteId: NoteInternalId): Promise<NoteSettings | null> {
```

Comment:
I think this method should not return "null". I see no sense in that, since there is no real case for such returning value.

It's better to throw an Error if update was not successful since it is a exception.

Reply:
Maybe throw error in patchNoteSettingsByNoteId? Or this method suppose to return null?